src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -981,6 +981,48 @@ void primitive_inst::do_runtime_skip_gather() {
     set_can_be_optimized(true);
 }
 
+void primitive_inst::do_runtime_skip_permute() {
+    OV_ITT_SCOPED_TASK(ov::intel_gpu::itt::domains::intel_gpu_plugin, openvino::itt::handle("do_runtime_skip_permute: " + id()));
+    // Check pattern
+    if (!get_node().is_type<permute>()
+        || is_output()
+        || !get_node().can_be_optimized()
+        || _impl_params->has_fused_primitives()
+        || _impl_params->get_input_layout(0).data_type != _impl_params->get_output_layout().data_type
+        || (get_user_insts().size() == 1 && get_users().front()->is_type<concatenation>() && get_user_insts().front()->can_be_optimized()))
```

Comment:
These  conditions are already covered by get_node().can_be_optimized() so I think they are not needed any more at runtime.

Reply:
Removed unnecessary condition